### ISSUE TITLE
[Feature Request] allow to modify the listening port/interface

### ISSUE THREAD

**Issue by anthosz** [im2nguyen/rover#24]
Hello,

I just discover this new tool that is wonderful!

But do you think it's possible to allow the modification of the port/IP (0.0.0.0 instead 127.0.0.1) via a parameter?

Can be useful in the case that we use the binary (instead docker) and the 9000 is already used by another stuff.

Best regards,

**Comment by HackyPenguin** [comment]
+1 This will also allow us to integrate this tool with tools like gitlab by creating a psudo review app to expose the changes. without the need to build a proxycontainer.

**Comment by im2nguyen** [comment]
This should be fixed with #32! Can you pull the latest Docker image and test it again? You can also re-download the binary 😄 

```
docker pull im2nguyen/rover
```

Then, run the following command to test it on any port you desire (this example is :8080)

```
docker run --rm -it -p 8080:8080 -v $(pwd):/src im2nguyen/rover -ipPort 0.0.0.0:8080
```

**Comment by anthosz** [comment]
Confirmed, thx!
